templates/zerver/development/design_testing/buttons.html

```diff
@@ -0,0 +1,171 @@
+{% extends "zerver/portico.html" %}
+{% set entrypoint = "dev-buttons" %}
+
+{% block title %}
+<title>Button Styles | Zulip Dev</title>
```

Comment:
Let's name this page something specific: "Button style preview" seems good, both here and on /devtools/